src/routes/articles.js

```diff
@@ -7,6 +7,9 @@ const {
 	ArticleDoesNotExistError,
 	TagAlreadyAssignedToPostError
 } = require("../services/errors")
+const validateSchema = require('../middleware/validateSchema')
+
+const validateRequest = validateSchema(true)
```

Comment:
so we'll be calling this function with the specific schema you need so you take this out.  import the specific schema you need

```js
const { updatePostSchema } = require('../schema')
```